polls/templates/polls/htmlform.html

```diff
@@ -9,24 +9,38 @@ <h3>Task</h3>
   <input type="hidden" value="{{ task_list }}" name="tasklist">
   <input type="hidden" value="{{ task_presentation }}" name="taskPresentation">
   <h4>ID: {{task.pk}}</h4>
-  <audio controls src="{{ reference_url }}" class="mt-3">
-    Your browser does not support the
-    <code>audio</code> element.
-  </audio>
-  <audio controls src="{{ transform_url }}" class="mt-3">
-    Your browser does not support the
-    <code>audio</code> element.
-  </audio>
-
-  <div class="mt-2">
-    <div class="form-check form-check-inline">
-      <input class="form-check-input" type="radio" id="type1" name="annotationOption" value="XXY" checked>
-      <label class="form-check-label" for="type1">First and Second is same</label>
+  <div>
+    {% for audio in audios %}
+
+    {% if forloop.counter == 1 %}
+    <div>
+
+      <audio controls src="{{ audio }}" class="mt-3">
+        Your browser does not support the
+        <code>audio</code> element.
+      </audio>
     </div>
-    <div class="form-check form-check-inline">
-      <input class="form-check-input" type="radio" id="type2" name="annotationOption" value="XYX">
-      <label class="form-check-label" for="type2">First and Third is same</label>
+
+    {% else %}
+    <audio controls src="{{ audio }}" class="mt-3">
+      Your browser does not support the
```

Comment:
I've seen this text in the if block and now in the else too, is there a condition that warrants that someone's browser wouldn't support the audio? and maybe we can suggest browsers that would work.

Reply:
It is just for triangular view of audios.